apps/src/lib/ui/LegacyDialogContents.js

```diff
@@ -68,6 +68,20 @@ export const LegacyTooFewDialog = () => (
   </ProtectedStatefulDiv>
 );
 
+export const LegacyIncorrectDialog = () => (
```

Comment:
The work to be able to use `BaseDialog` for this dialog would've blown up this PR. I created https://codedotorg.atlassian.net/browse/TEACH-515 to look at this in the future.